### DISCUSSION TITLE
Project selection drop-down is not displayed on Linux Debian

### DISCUSSION THREAD

**Discussion by fchampreux** [atom/atom#22616] [Q&A] [answered]
I work on several projects with Atom, on Windows and on Linux. On Windows, Atom displays this select box, on the top right of the screen, which very convenient to switch project or check wich project I am working on:  

![image](https://user-images.githubusercontent.com/1111213/122513177-5177fb00-d00a-11eb-8547-2d225b18ecdb.png)

But on Linux, I can't find how to activate this feature.
Does someone have the solution?
Thanks!

**Comment by DeeDeeG** [comment, marked as answer]
I have that dropdown in my Git tab on Ubuntu 20.04 using Atom 1.57

Do you have the Git tab open? You can click "Git" in the bottom-left corner to open the Git tab.

And then if that doesn't work you can try safe mode, which temporarily starts Atom without loading any addon packages. https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode

**Reply by fchampreux** [reply]
Thanks DeeDeeG !
I had an old version 1.38 without automated updates, and the feature did not exists yet.
Now it works perfectly !
